rootfs/etc/confd/templates/builder

```diff
@@ -136,11 +139,19 @@ kubectl --namespace=$POD_NAMESPACE create -f /etc/${SLUG_NAME}.yaml
 
 #check for image creation or slug existence in S3EP
 
-while [ ! -f /apps/${SLUG_NAME}/slug.tgz ]
-do
-  sleep 2
-  echo "no file"
-done
+if [[ "$REMOTE_STORAGE" == "1" ]]; then
+  LS_CMD="$MC_PREFIX ls $PUSH_URL"
+  until $LS_CMD &> /dev/null; do
+    echo "no file"
+    sleep 2
+  done
+else
+  while [ ! -f /apps/${SLUG_NAME}/slug.tgz ]
+  do
+    sleep 2
+    echo "no file"
+  done
```

Comment:
This is a good enough message for alpha. It's useful for debugging. We can clean it up later if we have time. Please don't spend time on UI elements until the builder is working.